projects/frontend/cicd/.gitlab-ci.yml

```diff
@@ -34,8 +34,24 @@ frontend-data-pipelines-build:
         - projects/frontend/data-pipelines/gui/reports/test-results/data-pipelines-lib/*.xml
     paths:
       - projects/frontend/data-pipelines/gui/package-lock.json
+      - projects/frontend/data-pipelines/gui/dist/
+      - projects/frontend/data-pipelines/gui/node_modules/
     expire_in: 1 week
 
+frontend-data-pipelines-e2e-tests:
+  image: "versatiledatakit/vdk-cicd-base-gui:2.0.0"
+  stage: pre_release_test
+  before_script:
+    - cd projects/frontend/
+  script:
+    - ./cicd/test_e2e_data_pipelines.sh "$VDK_API_TOKEN"
+  retry: !reference [.retry, retry_options]
+  rules:
```

Comment:
Consider generating and publishing npm test repo: 


```
artifacts:
    paths:
      - reports/ # Artifacts for the report
```
 
 to make it easier to debug / see issues in Gitlab website .

Reply:
I've separated the changes. Wiring the e2e tests goes in https://github.com/vmware/versatile-data-kit/pull/1754, where I've added e2e reports.